src/lang/std/sketch.ts

```diff
@@ -1174,29 +1174,31 @@ function isAngleLiteral(lineArugement: Value): boolean {
 
 type addTagFn = (a: ModifyAstBase) => { modifiedAst: Program; tag: string }
 
-function addTagWithTo(
-  argType: 'angleLength' | 'angleTo' | 'default' | 'length'
-): addTagFn {
+function addTag(): addTagFn {
```

Comment:
Not very DRY as it's cope paste from `giveSketchFnCallTag` except one takes a source range and the other pathToNode.